clustered/client/src/main/java/org/ehcache/clustered/client/internal/config/xml/ClusteringServiceConfigurationParser.java

```diff
@@ -183,12 +202,11 @@ private TimeoutDuration processGetTimeout(Element parentElement, Node timeoutNod
             String.format("Value of XML configuration element <%s> in <%s> exceeds allowed value - %s",
                 timeoutNode.getNodeName(), parentElement.getTagName(), amount));
       }
-      getTimeout = TimeoutDuration.of(amount.longValue(), convertToJavaTimeUnit(timeType.getUnit()));
+      return Duration.of(amount.longValue(), convertToJavaTimeUnit(timeType.getUnit()));
```

Comment:
This can now throw, same remark as before.

Reply:
Same comment :-) (ok, this one is not deprecated, but still, they will get a nice error)